Require by musl libc:
In file included from pgpolicies.c:9:0:
util.h:11:1: error: unknown type name ‘dev_t’
 dev_t parse_devt(const char *dev_t);

Cc: Christophe Varoqui <christophe varoqui opensvc com>
Cc: device-mapper development <dm-devel redhat com>
Signed-off-by: Xose Vazquez Perez <xose vazquez gmail com>
---
 libmultipath/util.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libmultipath/util.h b/libmultipath/util.h
index 257912c..8861085 100644
--- a/libmultipath/util.h
+++ b/libmultipath/util.h
@@ -1,6 +1,8 @@
 #ifndef _UTIL_H
 #define _UTIL_H
 
+#include <sys/types.h>
+
 size_t strchop(char *);
 int basenamecpy (const char * src, char * dst, int);
 int filepresent (char * run);
-- 
2.7.4
